src/python/foglamp/data_purge/sqlalchemy_insert.py

```diff
@@ -0,0 +1,67 @@
+"""
+Date: June 21 2017 
+Description: The following is the CREATE table and INSERT into table using SQLAlchemy
+            It is intended to be used for when showing how purge (currently) works example. 
+Lanague: Python + SQLAlchemy
+"""
+import sys
+import sqlalchemy
+import sqlalchemy.dialects
+import time
+
+# Set variables for connecting to database
+user="foglamp"
+db_user="foglamp"
+host="127.0.0.1"
+db="foglamp"
+
+# Create Connection
+engine = sqlalchemy.create_engine('postgres://%s:%s@%s/%s' % (db_user,user,host,db), pool_size=20, max_overflow=0)
+conn = engine.connect()
+
+# Important files
+configFile = 'config.json'
+logsFile = 'logs.json'
+
+readings = sqlalchemy.Table('readings',sqlalchemy.MetaData(),
+        sqlalchemy.Column('id',sqlalchemy.BIGINT,primary_key=True),
+        sqlalchemy.Column('asset_code',sqlalchemy.VARCHAR(50)),
+        sqlalchemy.Column('read_key',sqlalchemy.dialects.postgresql.UUID, default='00000000-0000-0000-0000-000000000000'),
+        sqlalchemy.Column('reading',sqlalchemy.dialects.postgresql.JSON,default='{}'),
+        sqlalchemy.Column('user_ts',sqlalchemy.TIMESTAMP(6),default=time.strftime('%Y-%m-%d %H:%M:%S',time.localtime(time.time()))),
+        sqlalchemy.Column('ts',sqlalchemy.TIMESTAMP(6), default=time.strftime('%Y-%m-%d %H:%M:%S',time.localtime(time.time()))))
+
+
+
+def execute_query(stmt):
+    """
+    EXECUTE query 
+    Args:
+        stmt: 
+    Returns:
+        largest ID
+    """
+    try:
+        result = conn.execute(stmt)
+    except:
+        print("SELECT Error")
```

Comment:
This is temporary because it will use the database service instead, but I would remove all of these try/except clauses and let exceptions get thrown. Don't exit when an exception occurs - let code "above" it catch exceptions. Otherwise, you won't be able to test failures.